performance-tests/src/test/jmeter/README.txt

```diff
@@ -0,0 +1,4 @@
+To run JMeter Test:
```

Comment:
Let's put this in the README.md file in the root of this project - that's where folks expect to find the instructions for using a project.